ui-core/src/main/java/com/adyen/checkout/components/internal/ui/view/RoundCornerImageView.kt

```diff
@@ -5,7 +5,7 @@
  *
  * Created by arman on 28/5/2019.
  */
-package com.adyen.checkout.components.ui.view
+package com.adyen.checkout.components.internal.ui.view
```

Comment:
Should this be restricted?

Reply:
No, it doesn't work if it's used in xml file